src/components/AppBody.jsx

```diff
@@ -140,6 +140,7 @@ class AppBody extends React.Component {
       setTimeout(() => parseReady = true, 1000);
       new Promise((resolve) => resolve(parser(snippet)))
       .then((AST) => {
+        console.log(JSON.stringify(AST));
```

Comment:
We need to be more cognizant of taking out our `console.log` statements in our commits.